src/test/java/org/alfresco/rest/api/tests/client/data/Node.java

```diff
@@ -86,6 +86,18 @@
     protected List<AssocTarget> targets;
 
     protected SearchResultEntry search;
+    
+    protected Boolean favorite;
```

Comment:
Please rename favorite to isFavorite so we're consistent with the naming convention for boolean in Node.